packages/loader/container-loader/src/container.ts

```diff
@@ -147,19 +147,33 @@ export interface IContainerConfig {
 
 export enum ConnectionState {
     /**
-     * The document is no longer connected to the delta server
+     * The container is not connected to the ordering service
+     * Note - When in this state the container may be about to reconnect,
+     * or may remain disconnected until explicitly told to connect.
      */
-    Disconnected,
+    Disconnected = 0,
 
     /**
-     * The document has an inbound connection but is still pending for outbound deltas
+     * The container is disconnected but actively trying to establish a new connection
+     * PLEASE NOTE that this numerical value falls out of the order you may expect for this state
      */
-    Connecting,
+    EstablishingConnection = 3,
 
     /**
-     * The document is fully connected
+     * @see ConnectionState.CatchingUp, which is the new name for this state.
+     * @deprecated - This state itself is not gone, just being renamed. Please use ConnectionState.CatchingUp.
```

Comment:
Does this need a breaking.md note?

Reply:
Yeah, coming soon™️  Thanks for taking a peek!

Reply:
See #10535